From b0ca355e118dd7d4bf147550fbce8ddd23140c8e Mon Sep 17 00:00:00 2001
From: anonym <anonym@riseup.net>
Date: Wed, 27 Feb 2019 09:44:54 +0100
Subject: [PATCH] Add comment for pref.

All other prefs in this section have comments, so not commenting this
one may even be confusing ("does the comment for
fetchFromExchange.enable also apply to guess.enabled?").
---
 comm/mailnews/mailnews.js | 3 +++
 1 file changed, 3 insertions(+)

--- a/comm/mailnews/mailnews.js
+++ b/comm/mailnews/mailnews.js
@@ -908,6 +908,9 @@ pref("mailnews.auto_config.fetchFromISP.
 // This also sends the email address and password to the server,
 // which the protocol unfortunately requires in practice.
 pref("mailnews.auto_config.fetchFromExchange.enabled", true);
+// Whether we will attempt to guess the account configuration based on
+// protocol default ports and common domain practices
+// (e.g. {mail,pop,imap,smtp}.<email-domain>).
 pref("mailnews.auto_config.guess.enabled", true);
 // Work around bug 1454325 by disabling mimetype mungling in XmlHttpRequest
 pref("dom.xhr.standard_content_type_normalization", false);
